### DISCUSSION TITLE
Some links say they're created when they aren't

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3700] [Error / Bug Report]
**Describe the bug**
I preemptively "made" every single link I have so I can easily click it and create it. This results in alternating "made" and "unmade" lines, despite no pages existing.

**To Reproduce**
1. Create test page.
2. Page links into markdown page.
3. Save.
4. Look at image.

**Expected behavior**
All links should be red.

**Screenshots**

![image](https://user-images.githubusercontent.com/57202889/69903179-ebf6e780-135b-11ea-9523-0f58e79739aa.png)

**Host Info (please complete the following information):**
Latest release.

**Comment by NGPixel** [comment]
Use relative paths, not absolute URLs, e.g.:

`/path/to/page`

**Comment by ghost** [comment]
@NGPixel They're all the same. They are relative paths.
